go/vt/orchestrator/db/generate_base.go

```diff
@@ -28,21 +28,21 @@ var generateSQLBase = []string{
           version varchar(128) CHARACTER SET ascii NOT NULL,
           binlog_format varchar(16) CHARACTER SET ascii NOT NULL,
           log_bin tinyint(3) unsigned NOT NULL,
-          log_slave_updates tinyint(3) unsigned NOT NULL,
+          log_replica_updates tinyint(3) unsigned NOT NULL,
           binary_log_file varchar(128) CHARACTER SET ascii NOT NULL,
           binary_log_pos bigint(20) unsigned NOT NULL,
-          master_host varchar(128) CHARACTER SET ascii NOT NULL,
-          master_port smallint(5) unsigned NOT NULL,
-          slave_sql_running tinyint(3) unsigned NOT NULL,
-          slave_io_running tinyint(3) unsigned NOT NULL,
-          master_log_file varchar(128) CHARACTER SET ascii NOT NULL,
-          read_master_log_pos bigint(20) unsigned NOT NULL,
-          relay_master_log_file varchar(128) CHARACTER SET ascii NOT NULL,
-          exec_master_log_pos bigint(20) unsigned NOT NULL,
-          seconds_behind_master bigint(20) unsigned DEFAULT NULL,
-          slave_lag_seconds bigint(20) unsigned DEFAULT NULL,
-          num_slave_hosts int(10) unsigned NOT NULL,
-          slave_hosts text CHARACTER SET ascii NOT NULL,
```

Comment:
These changes are a hard break: what happens to already existing `vtorc` setups? They will break and fail. Their table schema will not change, and suddenly the app expects different columns in the database.

Is there a migration path for those users? At the very least, they should be told they must delete their existing SQLite database. There should be docs explaining how to do so.

Reply:
I agree. We have not done any general release so far, so I thought we could do these changes without worrying about backward compatibility. I can change them to `ALTER TABLE` statements instead? We will need to add a documentation page anyways, since some of the configurations and the API endpoints have changed too

Reply:
>  I can change them to ALTER TABLE statements instead? 

FYI sqlite does not support `DROP COLUMN`. You can only add new columns

Reply:
We will need to write the proper release notes to warn people, but because vtorc is not GA, we are free to make breaking changes.
That is one of the reasons to announce a feature as experimental, it frees us from having to carry-forward code for backwards compatibility.